### PR TITLE
[14.0][ADD] maintenance_equipment_image

### PR DESCRIPTION
**Pull request by pedrocs-exo**
A module to add images to equipments

### CODE REVIEW

**Review by etobella — APPROVED** [review]
LGTM :+1:  Tested on runbot and code review. Check pre-commit  please

**Comment by pedrocs-exo** [comment]
@etobella pre-commit fixed. Thanks!

File: maintenance_equipment_image/tests/test_maintenance_equipment_image.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        f.seek(0)
+        sample_image = base64.b64encode(f.read())
+
+        self.Equipment = self.env["maintenance.equipment"]
+        self.equipment1 = self.Equipment.create(
+            {
+                "name": "Equipment 1",
+                "image": sample_image,
+            }
+        )
```
Comment (cubells) [inline maintenance_equipment_image/tests/test_maintenance_equipment_image.py]: What do you test in this test??

I think nothing.

**Review by cubells — CHANGES_REQUESTED** [review]

File: maintenance_equipment_image/tests/test_maintenance_equipment_image.py (hunk 1 above)
Comment (etobella) [reply maintenance_equipment_image/tests/test_maintenance_equipment_image.py]: Well, tests are allways welcome, but they are not correctly executed (some ifs are ignored) and in this case are not really needed. 

Also, make all in a single commit please

Comment (cubells) [reply maintenance_equipment_image/tests/test_maintenance_equipment_image.py]: If in this case are not really needed, then remove tests. Why have something that doesn't work?

Comment (pedrocs-exo) [reply maintenance_equipment_image/tests/test_maintenance_equipment_image.py]: Thank you for commenting. I removed the tests and squashed the commits into one by force pushing. Can you please check?

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by hitrosol — APPROVED** [review]
Tested in the runboat, LGTM

**Pull request merged**